frontend/src/modules/sidebar/less/sidebar.less

```diff
@@ -93,7 +93,8 @@
 }
 
 .sidebar-inner {
-  padding: 61px 0 25px 0;
+  padding: 0;
+  padding-bottom: 25px;
```

Comment:
Can we please remove `padding-bottom`? Then there's absolutely no scrollbar with a screen resolution of 1280 x 800 👍.

Reply:
I've tested with 800 as well and it worked for me. Did you have zoom applied?
Happy to remove the padding though 😄

Reply:
Don't be silly, I wouldn't use something as horrible as browser zoom 😉.

This is my Chrome window maximised on Windows 10 with a 1280 x 800 display:

![fullscreen](https://user-images.githubusercontent.com/922987/52488800-911bb600-2bb8-11e9-8bd4-e0f4065b5b66.jpg)